js/main.js

```diff
@@ -14,6 +14,17 @@ function rit(position) {
     var mylocation = "lat: " + position.coords.latitude +"<br>";
     mylocation += "lng: " + position.coords.longitude+"<br><br>";
     output.innerHTML = mylocation;
+    my_preview = "https://www.google.com/maps/?q=" + position.coords.latitude + "," +position.coords.longitude;
+    preview.innerHTML = "<a target='_blank' href='"+my_preview+"'>PREVIEW on GMaps!</a>"+"<br><br>";
+	var element = document.getElementById('osm-map');
+	element.style = 'height:300px;';
+	var map = L.map(element);
+	L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+        attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
```

Comment:
Any reason for using OpenStreetMap? You can use Google Maps instead

Reply:
@vaibhavsingh97 it's open source, we need to move away from priority software and encourage open source